diff --git a/meson.build b/meson.build
index 2eda9c64..9c07940c 100644
--- a/meson.build
+++ b/meson.build
@@ -101,11 +101,13 @@ endif
 
 vkd3d_build = vcs_tag(
   command : ['git', 'describe', '--always', '--exclude=*', '--abbrev=15', '--dirty=0'],
+  fallback: run_command('cat', files('BUILD')).stdout().strip(),
   input   : 'vkd3d_build.h.in',
   output  : 'vkd3d_build.h')
 
 vkd3d_version = vcs_tag(
   command : ['git', 'describe', '--always', '--tags', '--dirty=+'],
+  fallback: run_command('cat', files('VERSION')).stdout().strip(),
   input   : 'vkd3d_version.h.in',
   output  : 'vkd3d_version.h')
 
