diff --git a/meson.build b/meson.build
index 484c0c19..de42491e 100644
--- a/meson.build
+++ b/meson.build
@@ -80,7 +80,7 @@ idl_generator = generator(idl_compiler,
 glsl_compiler  = find_program('glslangValidator')
 glsl_generator = generator(glsl_compiler,
   output    : [ '@BASENAME@.h' ],
-  arguments : [ '--quiet', '-V', '--vn', '@BASENAME@', '@INPUT@', '-o', '@OUTPUT@' ])
+  arguments : [ '-V', '--vn', '@BASENAME@', '@INPUT@', '-o', '@OUTPUT@' ])
 
 threads_dep = dependency('threads')
 lib_d3d12   = vkd3d_compiler.find_library('d3d12', required : false)
